Tetrahedral_remeshing/include/CGAL/tetrahedral_remeshing.h

```diff
@@ -252,10 +252,54 @@ void tetrahedral_isotropic_remeshing(
                                  CGAL::parameters::all_default());
 }
 
+/*!
+* Converts the triangulation referenced in the input to a `Triangulation_3`.
```

Comment:
```suggestion
* Converts the triangulation contained in the input to a `Triangulation_3`.
```
"referenced" would mean that the c3t3 object detains a reference to an existing triangulation object, whereas the reality is that the triangulation is a (non-reference) data-member of the c3t3.

Reply:
right! I thought it was a reference